recipe/meta.yaml

```diff
@@ -36,16 +35,16 @@ test:
     - tableauserverclient.server
     - tableauserverclient.server.endpoint
   requires:
-    # Despite argparse being in pyproject.toml as optional for testing purposes, it's only used in
-    # the samples, not the tests.
-    # Including it makes Conda unable to resolve the environment properly, because argparse is so
-    # ancient at this point. This will likely need to be commented out/removed going forward.
-    # - argparse
-    - black
+    - pip
     - mock
-    - mypy
     - pytest >=7.0
     - requests-mock >=1.0,<2.0
+  source_files:
+    - test
+  commands:
+    - pip check
+    # ignore completely tests that depend on data files not contained in the tarball
+    - pytest test --ignore test/test_workbook.py --ignore test/test_view.py --ignore test/test_flow.py --ignore test/test_fileuploads.py --ignore test/test_filesys_helpers.py --ignore test/test_datasource.py
 
 about:
   home: https://github.com/tableau/server-client-python
```

Comment:
The source code also has a vendored `Versioneer` under the `CC0-1.0` license, see https://github.com/tableau/server-client-python/blob/v0.30/LICENSE.versioneer. And this license file should be included too

Reply:
addressed